projects/sensory_interface/public/app.js

```diff
@@ -66,6 +70,15 @@ function startSoundPlayback(selectedValue) {
     const MIN_FREQUENCY = 100;
     let minValue = document.getElementById("minValue").value;
     let maxValue = document.getElementById("maxValue").value;
+    maxValue = parseInt(maxValue);
```

Comment:
The value is not necessarily an int. If can be any fractional number.

Reply:
No problem! Will fix that.